src/node/genesisgen.h

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+#include "codeid.h"
+#include "entities.h"
+#include "luainterp/luainterp.h"
+#include "luainterp/luautil.h"
+#include "members.h"
+#include "networktables.h"
+#include "nodeinfonetwork.h"
+#include "nodes.h"
+#include "rpc/consts.h"
+#include "rpc/jsonrpc.h"
+#include "runtime_config/default_whitelists.h"
+#include "tls/keypair.h"
+#include "values.h"
+
+#include <algorithm>
+#include <fstream>
+#include <ostream>
+
+namespace ccf
+{
+  class GenesisGenerator
```

Comment:
What's a better name for this? It is mostly used by `nodestate.h` (and some of our tests) to read/write things to the KV without having to explicitly create a view for it, etc.

Reply:
As discussed, will leave as is and we will refactor this later.